backend/src/zango/cli/start_project.py

```diff
@@ -228,19 +228,32 @@ def start_project(
     # Creating Public Tenant
     create_public_tenant(platform_domain_url=platform_domain_url)
 
-    # Prompting default platform user details
-    if not platform_username:
-        click.echo("Please enter platform user email")
-        platform_username = click.prompt("Email")
-    if not platform_user_password:
-        platform_user_password = click.prompt(
-            "Password", hide_input=True, confirmation_prompt=True
+     # Prompting default platform user details
+    while True:
+        temp_username = platform_username
+        platform_username = None
+        platform_user_password = None
+
+        if not platform_username:
+            click.echo("Please enter platform user email")
+            platform_username = click.prompt("Email")
+        else:
+            click.echo("Please enter platform user email\nPrevious Attempt: " + temp_username)
+            platform_username = click.prompt("Email")
+        if not platform_user_password:
+            platform_user_password = click.prompt(
+                "Password", hide_input=True, confirmation_prompt=True
+            )
+
+        user_creation_result = create_platform_user(
+            platform_username, platform_user_password
         )
-
-    user_creation_result = create_platform_user(
-        platform_username, platform_user_password
-    )
-    if not user_creation_result["success"]:
-        click.echo("User Creation Failed!")
+        if user_creation_result["success"]:
+            break
+        else:
+            user_creation_result["message"]
```

Comment:
here we need to print this message also

Reply:
crap I thought I added that lol

Reply:
now it's added but, no need for line no. 254